### DISCUSSION TITLE
upgraded 2.3 now everything is https

### DISCUSSION THREAD

**Discussion by intsec** [Security-Onion-Solutions/securityonion#4108] [Unsupported Versions] [answered]
I upgraded a couple days ago and now my beats no longer can connect with http to elasticsearch.  Https has been turned on and I am not sure where the certificates are at to review what it needs for a hostname to connect to my SO.  These are for external devices.  Can I just turn off verifying certificate?

**Comment by TOoSmOotH** [comment]
Yes you should be able to just turn off verification and it should connect.

**Reply by intsec** [reply]
I tried looking on how to do this, I thought it was in the elasticsearch conf in /opt/so/conf but it didn't work.  Any thoughts?

**Reply by TOoSmOotH** [reply]
you would need to set that on the beat client

**Reply by intsec** [reply]
Ahh, ok.  I was kind of confused on that.  Thanks!

**Reply by intsec** [reply]
Ok, I have added ssl.enabled:false, ssl.verification_mode: none, ssl.client_authentication: none to the output of elasticserach in filebeat.yml and it still does it.  What am I missing?

**Comment by intsec** [comment, marked as answer]
Got it. I had to add the name of my SO hostname in my server running filebeat and then remove everything that I mentioned except for ssl.client_authentication: none in the output. It is now working. Thanks!
